### ISSUE TITLE
Protected Cookies hidden with new version of FF

### ISSUE THREAD

**Issue by BobinDC** [ysard/cookie-quick-manager#86]
I have a bunch of protected bookies that work well. I just loaded a new version of FF, and the protected cookies seemed to have disappeared. When I manage cookies, they aren't visible. But when I log back on to a website for which I have protected cookies, all the old protected cookies for that site reappear with the protection turned on. That's how I know that they aren't new cookies, because newly added cookies would not be protected. I'm not quite sure what's going on here, nor am I 100% sure that loading a new version of FF is the problem.

I've just learned (by reading other messages) about exporting and importing cookies, so that may provide a workaround for losing protected cookies.  But maybe there's an issue here that needs attention.

thanks for the add-on. It's important to me to have control over cookies.

Bob

**Comment by HassePL** [comment]
Obviously, there is no one who answers on this forum. I guess one cannot complain,. but it is strange, indeed.

Hans L
